runtime/kusama/src/lib.rs

```diff
@@ -616,10 +625,28 @@ parameter_types! {
 	pub const MaxKeys: u32 = 10_000;
 	pub const MaxPeerInHeartbeats: u32 = 10_000;
 	pub const MaxPeerDataEncodingSize: u32 = 1_000;
+	// PayOverXcmTimeout is set to 3 x the spend period to provide buffer before status messages
+	// are deleted from storage
+	pub const PayOverXcmTimeout: BlockNumber = 6 * 3 * DAYS;
+}
+
+ord_parameter_types! {
+	pub const TreasuryAccountId: AccountId = AccountIdConversion::<AccountId>::into_account_truncating(&TreasuryPalletId::get());
 }
 
 impl pallet_treasury::Config for Runtime {
 	type PalletId = TreasuryPalletId;
+	type AssetKind = xcm::latest::AssetId;
```

Comment:
with the setup there is no way to spend some DOTs on Statemint

Reply:
Since this is an xcm assetId, it can point to any asset at any xcm multilocation